app/src/main/java/hr/foi/air/sportloc/view/ui/fragment/ProfileViewFragment.java

```diff
@@ -133,13 +143,44 @@ public void ProfileDownvote() {
         downvote = true;
     }
 
-    public void reloadFragment(UserModel user) {
 
+    public void reloadFragment(UserModel user) {
         getArguments().clear();
         getArguments().putBoolean(Constants.RELOADED, true);
         getArguments().putSerializable(ModelEnum.UserModel.name(), user);
         getFragmentManager().beginTransaction().detach(this).attach(this).commit();
+        upvote = false;
+        downvote = false;
+    }
 
+    @OnClick(R.id.btn_comment)
+    public void writeComment() {
+        if (upvote == false && downvote == false) {
+            MessageSender.sendError(getContext(), getResources().getString(R.string.profile_write_error));
+        } else {
+            openWriteComment = true;
+            boolean userVote;
+            if (upvote == true) {
+                userVote = true;
+            } else {
+                userVote = false;
+            }
```

Comment:
wtf,
1. if(upvote == true) is actually if(upvote) there is no need for ==
2. userVote is actually upvote.... what is the purpose for that

Reply:
do you understand what this if and else actually do, you do not need userVote variable it serves no purpose what so ever....